kitsune/sumo/static/sumo/js/tests/form-wizard-setup-device-tests.js

```diff
@@ -107,26 +121,97 @@ describe("setup-device-step custom element", () => {
     it("should allow a valid email address to be submitted", async () => {
       expect([...emailErrorMessage.classList]).to.not.include("visible");
 
-      let preventSubmitListener = new Promise((resolve) => {
+      let submitListener = new Promise((resolve) => {
         form.addEventListener(
           "submit",
           (e) => {
-            e.preventDefault();
             resolve();
           },
           { once: true }
         );
       });
 
-      emailField.value = "example@example.com";
+      emailField.value = "invalid@invalid.invalid";
       emailField.dispatchEvent(new CustomEvent("input", { bubbles: true }));
       emailField.dispatchEvent(new CustomEvent("blur", { bubbles: true }));
       expect(submitBtn.disabled).to.be.false;
 
       expect(emailField.validity.valid).to.be.true;
       expect([...emailErrorMessage.classList]).to.not.include("visible");
       submitBtn.click();
-      await preventSubmitListener;
+      expect(submitBtn.disabled).to.be.true;
+      await submitListener;
+      expect(window.fetch.calledOnce).to.be.true;
+    });
+
+    it("should show an error message on a form submission error", async () => {
+      expect([...emailErrorMessage.classList]).to.not.include("visible");
+
+      let submitListener = new Promise((resolve) => {
+        step.addEventListener(
+          "DeviceMigrationWizard:SetupDeviceStep:EmailSubmitted",
+          (e) => {
+            resolve();
+          },
+          { once: true }
+        );
+      });
+
+      emailField.value = "invalid@invalid.invalid";
```

Comment:
```suggestion
      emailField.value = TEST_EMAIL_ADDRESS;
```

Maybe worth making this a constant so that it's more likely to get reused in future tests?

Reply:
Thanks! Done.